sqlalchemy_risingwave/base.py

```diff
@@ -113,7 +113,38 @@ def get_columns(self, conn, table_name, schema=None, **kw):
         return res
 
     def get_indexes(self, conn, table_name, schema=None, **kw):
-        return []
+        table_oid = self.get_table_oid(
+            conn, table_name, schema, info_cache=kw.get("info_cache")
+        )
+
+        sql = (
+            "select i.relname, a.attname, a.attnotnull from pg_catalog.pg_class t "
+            "join pg_catalog.pg_index ix on t.oid = ix.indrelid "
+            "join pg_catalog.pg_class i on i.oid = ix.indexrelid "
+            "join pg_catalog.pg_attribute a on t.oid = a.attrelid "
+            "where t.oid = :table_oid"
+        )
```

Comment:
Have you checked how Postgresql implement it? Is our apporach the same with them?

Reply:
PostgreSQL will add some attributes in the return value dictionary, but our return value is the same as cockroachdb. So sql is simpler than PostgreSQL. But there are some errors in this sql, I missed a condition in "on". I will fix it later.